crawler_mcp/crawlers/optimized/processing/result_converter.py

```
@@ -0,0 +1,514 @@
+            if (
+                hasattr(result, "media")
+                and result.media
+                and isinstance(result.media, dict)
+            ):
+                image_list = result.media.get("images", [])
+                if isinstance(image_list, list):
+                    for img in image_list:
+                        if isinstance(img, dict):
+                            src = img.get("src", "")
+                        else:
+                            src = str(img)
```

Ruff (0.12.2)

Use ternary operator `src = img.get("src", "") if isinstance(img, dict) else str(img)` instead of `if`-`else`-block

Replace `if`-`else`-block with `src = img.get("src", "") if isinstance(img, dict) else str(img)`

(SIM108)